src/modules/aws_vpc/mappers/route_table.ts

```diff
@@ -0,0 +1,266 @@
+import { EC2, paginateDescribeRouteTables } from '@aws-sdk/client-ec2';
+import { CreateRouteTableCommandInput } from '@aws-sdk/client-ec2/dist-types/commands/CreateRouteTableCommand';
+import {
+  Route as AwsRoute,
+  RouteTable as AwsRouteTable,
+} from '@aws-sdk/client-ec2/dist-types/models/models_1';
+
+import { AWS, crudBuilderFormat, paginateBuilder } from '../../../services/aws_macros';
+import { getCloudId } from '../../../services/cloud-id';
+import { findDiff } from '../../../services/diff';
+import { Context, Crud2, MapperBase } from '../../interfaces';
+import { Route, RouteTable } from '../entity';
+import { AwsVpcModule } from '../index';
+import { convertTagsForAws, convertTagsFromAws, eqTags } from './tags';
+
+export class RouteTableMapper extends MapperBase<RouteTable> {
+  module: AwsVpcModule;
+  entity = RouteTable;
+  equals = (a: RouteTable, b: RouteTable) => {
+    return (
+      a.vpc.vpcId === b.vpc.vpcId &&
+      this.eqListItems<Route>(this.eqRoute, a.routes, b.routes) &&
+      eqTags(a.tags, b.tags)
+    );
+  };
+
+  eqListItems<T>(eq: (a: T, b: T) => boolean, a?: T[], b?: T[]) {
+    const subset1 = !!a?.every(a1 => !!b?.find(a2 => eq(a1, a2)));
+    const subset2 = !!b?.every(a1 => !!a?.find(a2 => eq(a1, a2)));
+    return subset1 && subset2;
+  }
+
+  findRoutesDiff(dbEntities: Route[], cloudEntities: Route[]) {
+    const routeId = (route: Route) => {
+      const cloudColumns = getCloudId(Route) as string[];
+      return cloudColumns.map(col => (route as any)[col]).join('|');
+    };
+    const { entitiesInDbOnly, entitiesInAwsOnly } = findDiff(
+      dbEntities,
+      cloudEntities,
+      routeId,
+      () => true, // dummy function because all fields are cloud id
+    );
+    return { entitiesInDbOnly, entitiesInAwsOnly };
+  }
+
+  eqRoute(a: Route, b: Route) {
+    return (
+      a.DestinationCidrBlock === b.DestinationCidrBlock &&
+      a.DestinationIpv6CidrBlock === b.DestinationIpv6CidrBlock &&
+      a.DestinationPrefixListId === b.DestinationPrefixListId &&
+      a.EgressOnlyInternetGatewayId === b.EgressOnlyInternetGatewayId &&
+      a.GatewayId === b.GatewayId &&
+      a.InstanceId === b.InstanceId &&
+      a.InstanceOwnerId === b.InstanceOwnerId &&
+      a.NatGatewayId === b.NatGatewayId &&
+      a.TransitGatewayId === b.TransitGatewayId &&
+      a.LocalGatewayId === b.LocalGatewayId &&
+      a.CarrierGatewayId === b.CarrierGatewayId &&
+      a.NetworkInterfaceId === b.NetworkInterfaceId &&
+      a.VpcPeeringConnectionId === b.VpcPeeringConnectionId &&
+      a.CoreNetworkArn === b.CoreNetworkArn
+    );
+  }
```

Comment:
You could probably replace this whole thing with [`_.isEqual`](https://docs-lodash.com/v4/is-equal/) which we use in a couple of other modules.